BlueprintUI/Sources/Layout/Overlay.swift

```diff
@@ -5,15 +5,28 @@ import UIKit
 /// During a layout pass, measurement is calculated as the max size (in both x and y dimensions)
 /// produced by measuring all of the child elements.
 ///
-/// View-backed descendents will be z-ordered from back to front in the order of this element's
+/// View-backed descendants will be z-ordered from back to front in the order of this element's
 /// children.
 public struct Overlay: Element {
 
+    /// All elements displayed in the overlay.
     public var elements: [Element]
 
-    public init(elements: [Element]) {
+    /// Creates a new overlay with the provided elements.
+    public init(
+        elements: [Element] = [],
+        configure : (inout Overlay) -> () = { _ in}
```

Comment:
I kinda feel like this would make more sense as two separate inits.

Reply:
I thought about that, but they do get matrixed out when autocompleting (eg you get to choose from the one with or without methods), so it felt fine to keep it as one.